src/components/Button/Button.scss

```diff
@@ -0,0 +1,81 @@
+$offset: 187;
+$duration: 1.4s;
+
+.reactive-button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: #fff;
+  border: none;
+  outline: 0;
+  transition: all 0.8s;
+  font-size: 0.9em;
+  font-weight: 500;
+  padding: 7px 18px;
+  border-radius: 3px;
+  background-color: #4cade8;
+  background-position: center;
+  box-shadow: 0 3px 10px rgba($color: #4cade8, $alpha: 0.3);
+
+  .loading {
+    padding-left: 10px;
+  }
+
+  &:focus {
+    outline: 0 !important;
+  }
+  &:hover {
+    cursor: pointer;
+    background: #68b8e8 radial-gradient(circle, transparent 1%, #68b8e8 1%)
+      center/15000%;
+  }
+  &:active {
+    background-color: #4cade8;
+    background-size: 100%;
+    transition: all 0s;
+  }
+
+  .spinner {
+    animation: rotator $duration linear infinite;
+  }
+
+  @keyframes rotator {
+    0% {
+      transform: rotate(0deg);
+    }
+    100% {
+      transform: rotate(270deg);
+    }
+  }
+
+  .path {
+    stroke-dasharray: $offset;
+    stroke-dashoffset: 0;
+    stroke: #fff;
+    transform-origin: center;
+    animation: dash $duration ease-in-out infinite;
+  }
+
+  @keyframes dash {
+    0% {
+      stroke-dashoffset: $offset;
+    }
+    50% {
+      stroke-dashoffset: $offset/4;
+      transform: rotate(135deg);
+    }
+    100% {
+      stroke-dashoffset: $offset;
+      transform: rotate(450deg);
+    }
+  }
+}
+
+.error {
```

Comment:
Is this error only applicable to buttons? If not, it should be nested to avoid overwriting